inst/components/sidebar.scss

```diff
@@ -158,6 +158,10 @@ $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid
     --bslib-collapse-toggle-transform: -180deg;
     --bslib-collapse-toggle-right-transform: -180deg;
 
+    &[data-sidebar-init-auto-collapse] {
```

Comment:
Add a comment about this is neede for `open = "desktop"` and is related to this JS that reads the CSS var